README.md

```diff
@@ -179,22 +179,7 @@ Rust abstractions over the lower level WebAssembly interface.
 
 ## Oak Node
 
-An **Oak Node** is an instance of an Oak Module together with a policy
-configuration, running on an Oak VM, potentially within an enclave.
-
-Once a new Oak Node is initialized and its endpoint available, one or more
-clients (according to the policy configuration) connect to it using individually
-end-to-end encrypted, authenticated and attested channels. The remote
-attestation process proves to the client that the remote enclave is indeed
-running a genuine Oak VM and will therefore obey the policies set on the Oak
-Node; the Oak VM itself may then optionally prove additional details about the
-Oak Module and its properties, which may require reasoning about its internal
-structure.
-
-The Oak Module and the policies associated with an Oak Node are established once
-and for all at the time the Oak Node is created by the Oak Manager, and they
-cannot be modified once the Oak Node is running. Therefore each client only
-needs to verify the attestation once before it starts invoking the Oak Node.
+An **Oak Node** is an instance of an Oak Module running on an Oak VM.
```

Comment:
So we have 1:1 Module:Node and n:1 Nodes/Modules:Application? Do we really need the distinction between a node and a module? (similar to object/instance?)

Reply:
I am using Module to refer to the code, and Node to the instance. In this terminology, developers write Modules, not Nodes. But if that's confusing we can drop the term Module too and just say that developers write Nodes, even if it's not super accurate. Thoughts?